frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunCustomDetails.tsx

```diff
@@ -57,6 +58,7 @@ const PipelineRunCustomDetails: React.FC<PipelineRunCustomDetailsProps> = ({ pip
         namespace={pipelineRun.metadata.namespace}
         ownerResourceName={pipelineRun.metadata.name}
       />
+      <PipelineRunRepositories pipelineRun={pipelineRun} />
```

Comment:
A pipeline Run can associate back to only one repository, the name here needs to be changed. maybe `RepositoryLinkList` ?

Reply:
renamed